### ISSUE TITLE
[Doc][Wiki] Duplicated entries in Wiki and documentation page

### ISSUE THREAD

**Issue by loumalouomega** [KratosMultiphysics/Kratos#12908]
**Description**

Hello, right now we are redictedting both to the wiki (https://github.com/KratosMultiphysics/Kratos/wiki) and the doc page (https://kratosmultiphysics.github.io/).

I think we should purge the Wiki (maybe linking to the doc page suring transition) and eventually remove the wiki as it has duplicated entries.

@KratosMultiphysics/technical-committee FYI

**Comment by roigcarlo** [comment]
Agree, but first we need to migrate all the content from the wiki to the docs pages, which is the painful part.

**Comment by loumalouomega** [comment]
> Agree, but first we need to migrate all the content from the wiki to the docs pages, which is the painful part.

True, IMHO we should replace one by one and link to the new one in the old one (removing the content), once everything is migrated we can safely remove it.
